features/cache.feature

```diff
@@ -148,6 +148,10 @@ Feature: Managed the WordPress object cache
       Warning: Ignoring the --url=<url> argument because flushing the cache affects all sites on a multisite installation.
       """
 
+  @require-wp-6.1
+  Scenario: Checking if the cache supports a feature
+    Given a WP install
+
```

Comment:
@danielbachhuber I don't understand what use this has.
`non_existing` doesn't exist by default.
Wouldn't it be better to run a test on 6.1 for a feature that is defined (in 6.2)

Reply:
> I don't understand what use this has.

@janw-me `@require-wp-6.1` means the test will only run on WordPress 6.1 and higher. WordPress 3.7 is a part of the testing matrix, so an earlier test on this branch failed: https://github.com/wp-cli/cache-command/actions/runs/4620028330/jobs/8169572449

> `non_existing` doesn't exist by default.
> Wouldn't it be better to run a test on 6.1 for a feature that is defined (in 6.2)

I think the test is fine as it is. It doesn't need to be overly complex.